Can't build from source download

If you use the GitHub feature to download the source as a zip, perhaps because you don't want to have a dependency on git, Uppend won't build. It looks like it tries to use git to set the version in the build. Here's the log:

```
$ ./gradlew build
Downloading https://services.gradle.org/distributions/gradle-3.3-all.zip

<snip>

FAILURE: Build failed with an exception.

* Where:
Build file '/home/ec2-user/uppend-master/build.gradle' line: 17

* What went wrong:
A problem occurred evaluating root project 'uppend'.
> Cannot find '.git' directory

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 19.604 secs
```

Should this be supported, or is the answer in this case to download the jar from maven?

--- a/build.gradle
+++ b/build.gradle
@@ -14,7 +14,12 @@ plugins {
 
 import org.apache.tools.ant.filters.ExpandProperties
 
-version gitVersion()
+if (new File('.git').exists()) {
+    version gitVersion()
+} else {
+    version "unspecified"
+}
+
 
 group 'com.upserve'
 

